docs/assemblies/creating-the-data-plane.adoc

```diff
@@ -21,6 +21,8 @@ To create and deploy a data plane, you must perform the following tasks:
 
 * A functional control plane, created with the OpenStack Operator.
 * Pre-provisioned nodes must be configured with an SSH public key in the `$HOME/.ssh/authorized_keys` file for a user with passwordless `sudo` privileges.
+* Pre-provisioned nodes must be configured with an FQDN. An FQDN is configured in `/etc/hosts` using an IP of the node, and a short hostname with domainname. The line in `/etc/hosts` looks like:
```

Comment:
We don't use /etc/hosts in OSP18. Please check my related comment in the edpm-ansible PR.